#base_frame: "/base_laser_front_link"
base_frame: "/robot_0/base_laser_link"
segmentation:
    type: "AdaptiveThresholdDetection"
    angle: 0.2
    sigma: 0.2
filtering:
  min_points: 5
  max_points: 100
  min_area: 0.005
  max_area: 1.0
  min_dimension: 0.05
feature_extraction:
  min_angle_between_lines: 0.2
  max_distance: 0.05
  rho_resolution: 0.05
  theta_resolution: 0.04
  voting_threshold: 10
tracking:
  time_step: 0.1
  max_mahalanobis_distance: 1.0
  skip_decay_rate: 20.0
  probability_start: 0.01
  probability_detection: 0.999
  measurement_noise_covariance: [0.1, 0.0,
                                 0.0, 0.1]
  initial_state_covariance: [0.1, 0.0, 0.0, 0.0,
                             0.0, 0.1, 0.0, 0.0,
                             0.0, 0.0, 0.1, 0.0,
                             0.0, 0.0, 0.0, 0.1]
  process_noise_covariance: [0.2, 0.0, 0.0, 0.0,
                             0.0, 0.2, 0.0, 0.0,
                             0.0, 0.0, 0.2, 0.0,
                             0.0, 0.0, 0.0, 0.2]
  mean_false_alarms: 0.00002
  max_depth: 1
  min_g_hypothesis_ratio: 0.001
  max_g_hypothesis: 100